ui-tests/src/test/java/org/openmrs/reference/FindPatientRecordTest.java

```diff
@@ -33,11 +34,12 @@ public void deletePatient() throws Exception {
         deletePatient(patient);
     }
 
-    @Ignore
-    @Category(org.openmrs.reference.groups.BuildTests.class)
-    public void testFindPatientRecord() throws InterruptedException {
+    @Test
+    @Category(BuildTests.class)
+    public void findPatientRecordTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
+        findPatientPage.waitForPageToLoad();
```

Comment:
`Waiting the page to load` ensures that the entered patient identifier matches with the created test patient identifier. Otherwise without` waiting the page to load`, the comparison is made against the searched patient identifier with any identifier (`randomly picked from the db pool`) of the existing patient data present in the  `db server instance` because the `test patient data` is not saved in the `db server` instance